### PR TITLE
Remove useless bootstrap script in hive docker files

### PR DESCRIPTION
**Pull request by mbenguig**

### CODE REVIEW

File: hive/Dockerfile [hunk 1]
```diff
@@ -30,8 +30,4 @@ ADD hive_sql.hql $HIVE_HOME/
 
 WORKDIR $HIVE_HOME
 
-ADD bootstrap.sh /etc/bootstrap.sh
-RUN chmod +x /etc/bootstrap.sh
-
-
-CMD ["/etc/bootstrap.sh", "-d"]
\ No newline at end of file
+CMD ["/bin/bash"]
```
Comment (fviale) [inline hive/Dockerfile:33]: this change is strange as your container does not start hive at all

**Comment by fviale** [comment]
@mbenguig I don't understand your change

File: hive/Dockerfile (hunk 1 above)
Comment (mbenguig) [reply hive/Dockerfile:33]: Hive is started by the task and need some config files to be instanciated by the task before starting. Therefore, we cannot start hive at the docker run level via a bootstrap script. And i decided to follow the same logic as for hdfs-spark.

Comment (fviale) [reply hive/Dockerfile:33]: ok

**Review by fviale — APPROVED** [review]

**Pull request merged**